resources/migrations/000_migrations.yaml

```diff
@@ -8369,6 +8369,21 @@ databaseChangeLog:
                   remarks: 'Nullable column to incidate collection''s authority level. Initially values are "official" and nil.'
                   constraints:
                     nullable: true
+
+
+  - changeSet:
+      id: 310
+      author: howonlee
+      comment: Added 0.40.0 Migrate friendly field names
+      changes:
+        - update:
+            tableName: setting
+            columns:
+              - column:
+                  name: value
+                  value: 'simple'
+            where: "'key'='humanization-strategy' AND value='advanced'"
```

Comment:
I think this is broken

```sql
WHERE 'key' = 'humanization-strategy'
```

is comparing two string literals that will never be equal. This should have been `key`, not `'key'`

Reply:
It is wrong, and was fixed in changeset 311 right after #16482

Reply:
Oh right, forgot about that. We probably should have removed the old one or at least added a comment to it at the time.

Let's do that now so it doesn't trip any more people up